src/MensaWidget/MWFavorites.class.st

```diff
@@ -0,0 +1,66 @@
+"
+A MWFavorites is a class that stores all liked foods and also handle the liking process.
```

Comment:
For consistency, it should be ‘…liked foods…’

Reply:
i assumed you meant meals

Reply:
do you mean "liked meals"?